huxley/logs/management/commands/erase_logs.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+
+from huxley.logs.models import LogEntry
+
+class Command(BaseCommand):
+    args = '<days> (optional).'
+    help = 'Deletes all log entries after a specified number of days (defaults to 30).'
+
+    def handle(self, *args, **kwargs):
+        if args:
+            try:
+                days = int(args[0])
+            except ValueError as e:
+                raise e
```

Comment:
If we're just gonna raise the error again, may as well not even `try/except` it. Just let it happen.
```python
days = int(args[0]) if args else 30
```